nightwatch.conf.js

```diff
@@ -12,6 +14,92 @@ if (isRepoTest) {
   srcFolders = 'tests/nightwatch/';
 }
 
-const config = nightwatchConfig(webpackConfig, srcFolders);
+const port = 8080;
+const seleniumPort = 4444;
+const seleniumHost = ip.address();
+
+// eslint-disable-next-line no-console
+console.warn('WARNING: WebdriverIO should be used instead of Nightwatch.  Nightwatch will be deprecated in the future.');
+
+const seleniumDocker = new SeleniumDockerService();
+const expressDevService = new ServeStaticService();
+
+const startDriverAndServer = (done) => {
+  const webPackPromise = expressDevService.onPrepare({
+    webpackConfig,
+    serveStatic: { port },
+  });
+
+  const dockerPromise = seleniumDocker.onPrepare({
+    host: seleniumHost,
+    port: seleniumPort,
+    path: '/wd/hub',
+    seleniumDocker: {
+      enabled: !process.env.TRAVIS && !process.env.CI,
+    },
+  }, [{ browserName: 'chrome' }]);
+
+  Promise.all([webPackPromise, dockerPromise]).then(done);
+};
+
+const stopDriverAndServer = (done) => {
+  Promise.all([
+    expressDevService.onComplete(),
+    seleniumDocker.onComplete(),
+  ]).then(done);
+};
+
+const endBrowserSession = (browser, done) => browser.end(done);
+
+const config = {
+  selenium: {
+    start_process: false,
+  },
+  src_folders: srcFolders,
+  output_folder: 'reports',
+  custom_commands_path: '',
+  custom_assertions_path: '',
+  page_objects_path: '',
+  persist_globals: true,
+  test_workers: false,
+  detailed_output: !process.env.TRAVIS && !process.env.CI,
+  test_settings: {
+    default: {
+      launch_url: `http://${seleniumHost}:${port}`,
+      persist_globals: true,
+      selenium_port: seleniumPort,
+      selenium_host: process.env.TRAVIS ? 'standalone-chrome' : seleniumHost,
```

Comment:
This is the only line that is different from terra-toolkit.  I'm ok with moving everything here as nightwatch is going away eventually.  Thoughts?

Reply:
Just for posterity, I moved this code back to terra-toolkit.